src/codec_svt.c

```diff
@@ -125,14 +129,18 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         svt_config->is_16bit_pipeline = image->depth > 8;
 #endif
 
+#if !SVT_AV1_CHECK_VERSION(1, 5, 0)
```

Comment:
I verified that in SVT-AV1 v1.5.0, the comment for `high_dynamic_range_input` changed from `Enable writing of HDR metadata in the bitstream` to `Currently unused`. See https://gitlab.com/AOMediaCodec/SVT-AV1/-/commit/33e0fdfe3.

But even in SVT-AV1 v1.0.0, `high_dynamic_range_input` is unused. So I suggest we test `!SVT_AV1_CHECK_VERSION(3, 0, 0)` instead here.

Reply:
done